File: src/main/java/io/teipub/pub/domain/Post.java
package io.teipub.pub.domain;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import java.time.LocalDateTime;

/**
 * Created by tei on 17. 3. 15.
 * mac-sys.co.kr
 * roryalroad.co.kr
 */
@Data
@Entity
public class Post {
    @Id
    @GeneratedValue
    private Integer id;
    private String title;
    private String content;
    private LocalDateTime createdAt;
}

File: src/main/kotlin/io/teipub/pub/domain/Post.kt
package io.teipub.pub.domain

import java.util.*
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.Id

/**
 * Created by tei on 17. 3. 3.
 * mac-sys.co.kr
 * roryalroad.co.kr
 */
@Entity
class Post {
    @Id
    @GeneratedValue
    var id: Int? = null

    lateinit var title: String
    lateinit var content: String
    lateinit var createdAt: Calendar
}